orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/host/HostOrchestrator.java

```diff
@@ -128,6 +128,10 @@ Map<String, Map<String, String>> formatAndMountDisksOnNodesLegacy(List<GatewayCo
     void backupDatabase(GatewayConfig primaryGateway, Set<String> target, SaltConfig saltConfig, ExitCriteriaModel exitModel, int databaseMaxDurationInMin)
```

Comment:
isnt it better adding the compress parameter here ?

Reply:
yes, combined the calls into one